internal/api/http/parser.go

```diff
@@ -26,3 +27,24 @@ func (p *Parser) Parse(r *http.Request, data interface{}) error {
 
 	return p.decoder.Decode(data, r.Form)
 }
+
+func (p *Parser) UnwrapParserError(err error) error {
```

Comment:
If you do an Unwrap or errors, please also provide unit tests on what responses should be expected.